build-tools/create-packs/Directory.Build.targets

```diff
@@ -124,6 +124,7 @@
       <_NuGetSources Include="$(OutputPath.TrimEnd('\'))" />
       <_PreviewPacks Condition=" '$(AndroidLatestStableApiLevel)' != '$(AndroidLatestUnstableApiLevel)' " Include="$(XamarinAndroidSourcePath)bin\Build$(Configuration)\nuget-unsigned\Microsoft.Android.Ref.$(AndroidLatestUnstableApiLevel).*.nupkg" />
       <_InstallArguments Include="android" />
+      <_InstallArguments Include="android-net8" />
```

Comment:
I wonder if we can have a better name for this workload? 
Given it is a moving target, I can imagine for .net 12 we would need a `android-net10` package for a while.

Problem is I can't think of anything lol.

Reply:
They have `wasm-tools-net8`:

https://github.com/dotnet/runtime/blob/953bc61ba9fc14dd1031ce83fc2700c7129042ac/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net8.Manifest/WorkloadManifest.json.in#L7

Some of the Mono ones say `-net8`, too.

So, I think maybe `android-36` is the outlier? Maybe it should be named `android-api36` to reduce confusion?